app/api-requests/rpa-api/base.js

```diff
@@ -0,0 +1,33 @@
+const wreck = require('@hapi/wreck')
+const session = require('../../session')
+const { tokens } = require('../../session/keys')
+const config = require('../../config')
+const apiHeaders = require('../../constants/api-headers')
+
+const get = async (hostname, url, request, headers = {}) => {

```

Comment:
Would this be all right to be named something more descriptive?

Reply:
This is a generic GET method that can be used for all calls to the RPA host but with different URL endpoints. I think this name is good enough, it would also be difficult to come up with anything descriptive but also generic enough (unless you have any thoughts) ;)